blocks/form/form.css

```diff
@@ -394,19 +395,13 @@ main .study-request .form-submission-type-fs {
     --form-fieldset-legend-font-weight: bold;
 }
 
-main .study-request :is(.form-authorize, .form-terms, .form-submit-wrapper) {
-    width: 500px;
-    margin: auto;
-    text-align: center;
-    font-style: oblique;
-}
-
 @media (min-width: 640px) {
     main .connectus {
         --form-columns: 6;
     }
 
-    main .connectus :is(.form-postalcode, .form-country, .form-phone, .form-email, .form-firstname, .form-lastname) {
+    /* stylelint-disable-next-line  selector-class-pattern */
```

Comment:
We have to disable this eslint rule to support name in camelCase, The eslint support kebab-case.

Reply:
@solaris007 - FYI, we had to change name for sync data into Marketo